src/style.css

```
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.searchBarBig {
+  margin: 20px 0;
+}
```

you need margins on both forms, don't use `big` or `first` classes when you can do this
```suggestion
.searchBar {
  margin-top: 20px;
}
```